srcpkgs/ngx_headers_more/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'ngx_headers_more'
+pkgname=ngx_headers_more
+version=0.33
+revision=1
+_nginx_version=1.18.0
+wrksrc=nginx-${_nginx_version}
+build_style=gnu-makefile
+make_build_args="modules"
+hostmakedepends="pcre-devel"
+makedepends="${hostmakedepends}"
```

Comment:
I think we can remove hostmakedepends and leave pcre-devel in makedepends.

Reply:
Removing makes cross-compiling fail